src/unity/lib/annotation/CMakeLists.txt

```diff
@@ -3,9 +3,14 @@ project(unity)
 set(TC_ANNOTATIONS_PROTO_FILES
   build/format/cpp/annotate.pb.cc
   build/format/cpp/data.pb.cc
+  build/format/cpp/message.pb.cc
   build/format/cpp/meta.pb.cc
 )
 
+set(TC_ANNOTATIONS_CLASS_REGISTRATION_FILE
+  class_registrations.cpp
```

Comment:
This can probably just go in the list of SOURCE_FILES unless it's used separately elsewhere.

Reply:
Fixed!